desc/compute/_stability.py

```diff
@@ -234,199 +236,191 @@ def _magnetic_well(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="c ballooning",
```

Comment:
do we want these as part of the public list of variables? Did we ever add the option to register a function but leave it private?

Reply:
that sounds like we want it public. can remove the `public=False` in the compute function register before this PR is merged